### ISSUE TITLE
App crashed when use SearchBar 

### ISSUE THREAD

**Issue by PramodMAUI** [dotnet/maui#14316]
### Description

Created sample MAUI project and used Searchbar control in MainPage.xaml and run on iOS(12) phyical device, App is getting crashed.
In output window of Visual Studio 2022 - "The app has been terminated."
Sample project has been placed here: https://github.com/PramodMAUI/TestMAUI

### Steps to Reproduce

Just create a new sample with .NET7 and add SearchBar control in MainPage.xaml. App will crash on launch.

### Link to public reproduction project repository

https://github.com/PramodMAUI/TestMAUI

### Version with bug

6.0.312

### Last version that worked well

6.0.312

### Affected platforms

iOS

### Affected platform versions

iOS 12

### Did you find any workaround?

No. 

### Relevant log output

```shell
The app has been terminated.
Sample project has been placed here: https://github.com/PramodMAUI/TestMAUI
```

**Comment by PramodMAUI** [comment]
Hi,

I have checked the same sample (https://github.com/PramodMAUI/TestMAUI) on Visual Studio Version 17.6.0 Preview 2.0. Still, I'm facing the same issue.
App is crashing on launch when we use Searchbar control.
It is working fine on iPad pro (iOS 16.2) but crashed on iOS 12.

**Comment by TamerIrdais** [comment]
I am having the same issue on VS 2022 17.5.4 when running on iPad 12.5.7 but its NOT active on iPhone 12 Pro Max (iOS 15.6.1).

**Comment by PramodMAUI** [comment]
Any update on this? Still, we are facing crash on SearchBar in iOS 12 versions.

**Comment by TamerIrdais** [comment]
I also tried this issue on latest officially released VS 17.6 and its still active, we need an update on this issue please.

**Comment by PramodMAUI** [comment]
This is pending from a long. Can we have any update on this?

**Comment by cybsaurabhBa** [comment]
Facing same issue with search bar. Is any update on this issue?

**Comment by pratiksalv** [comment]
App crash present on iOS 12 device due to the search bar. Can we get this fix asap?

**Comment by chucker** [comment]
From skimming the code, I would say 8509b26 introduced an infinite loop on iOS < 13, where GetSearchTextField now keeps calling itself, when instead it should be calling FindDescendantView<UITextField>().

**Comment by kubaflo** [comment]
Closing as no longer applicable: this report targets an old iOS 12-specific crash scenario. If you can still reproduce similar behavior on a supported iOS version, please open a fresh issue with a current repro.

**Comment by kubaflo** [comment]
Reopening — this issue should not have been closed.

It carries `s/verified` (a maintainer or partner reproduced it), `s/triaged` (it already went through triage). Those labels mean the report was already confirmed as a real product defect, so it is not eligible for backlog cleanup closure. Closure here requires specific fix evidence (a merged PR that closes it, or a `fixed-in-*` label), and there was none.

Sorry for the noise, and thanks to everyone who flagged it.

> Disclosure: this comment was generated by AI.

**Comment by chucker** [comment]
It seems as of .NET 11 preview, [iOS 12 is still considered supported](https://learn.microsoft.com/en-us/dotnet/maui/supported-platforms?view=net-maui-11.0):

>iOS 12.2 or higher is required.

If those docs are indeed up-to-date, the issue shouldn't be closed.

**Comment by kubaflo** [comment]
You're right, and this is already sorted — I reopened the issue at 12:30 UTC today, shortly before your comment landed.

Your point is confirmed: the .NET MAUI 11 supported-platforms doc does still list **iOS 12.2 or higher** as the minimum, so the "targets an old iOS 12-specific scenario, no longer applicable" rationale I used was simply wrong. On top of that, this issue carries `s/verified` and `s/triaged`, which means it was already confirmed as a real defect and should never have been part of a bulk cleanup pass at all.

It stays open. Thanks for catching it.

> Disclosure: this comment was generated by AI.
